india_compliance/gst_india/setup/property_setters.py

```diff
@@ -35,6 +35,21 @@ def get_property_setters(*, include_defaults=False):
             ["Bill of Entry"],
             prepend=False,
         ),
+        get_options_property_setter(
```

Comment:
Do we need this?

Reply:
GST Invoice number cannot exceed 16 characters.